lib/Client/UnitTestClient.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Omise\Client;
+
+use Exception;
+
+class UnitTestClient implements ClientInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function setCredential($key)
+    {
+        return;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function execute($url, $method, $params)
+    {
+        $fixtureDirectory = __DIR__ . '/../../tests/fixtures';
+        $requestUrl       = $this->extractRequestUrl($url);
+
+        $fixture = $fixtureDirectory . '/' . $requestUrl . '-' . strtolower($method) . '.json';
+        if (! file_exists($fixture)) {
+            throw new Exception('Fixture not found: ' . $fixture);
```

Comment:
Note, here I intended to remove the auto-generate fixture file out from this method as I think that auto-generate function should be in a test script instead of a production code.